2022-01-12 00:16:24,557 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 569625ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 00:16:24,539 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 00:26:36,539 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 571027ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 00:26:36,453 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 00:36:50,550 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 570821ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 00:36:50,535 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 00:47:02,658 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 573491ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 00:47:02,524 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 00:57:15,401 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 00:57:15,865 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 570380ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 01:07:27,666 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 01:07:27,676 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 570880ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 01:17:39,516 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 570871ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 01:17:39,498 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 01:27:52,662 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 571605ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 01:27:52,609 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 01:38:04,536 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 01:38:04,461 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 571361ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 01:48:16,530 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 571730ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 01:48:16,454 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 01:58:27,383 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 01:58:27,608 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 569876ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 02:08:40,959 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 02:08:40,614 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 570964ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 02:18:54,497 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 02:18:54,513 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 572475ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 02:29:04,504 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 02:29:04,557 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 572055ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 02:39:18,422 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 570626ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 02:39:18,391 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 02:49:29,607 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 570966ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 02:49:29,517 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 02:59:41,491 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 02:59:41,457 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 569747ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 03:09:53,556 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 570837ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 03:09:53,651 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 03:20:04,560 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 03:20:04,707 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 569826ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 03:30:17,550 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 571388ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 03:30:17,568 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 03:40:29,553 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 03:40:29,470 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 570725ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 03:50:40,567 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 03:50:40,504 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 570534ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 04:00:53,608 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 571572ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 04:00:53,509 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 04:11:06,519 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 572430ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 04:11:06,537 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 04:21:17,422 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 571395ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 04:21:17,470 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 04:31:28,085 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 570053ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 04:31:27,534 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 04:41:40,530 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 570833ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 04:41:40,513 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 04:51:52,482 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 04:51:52,462 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 571185ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 05:01:59,506 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 05:01:59,568 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 570160ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 05:12:16,647 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 571945ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 05:12:16,642 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 05:22:29,325 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 571520ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 05:22:29,409 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 05:32:41,521 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 571463ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 05:32:41,478 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 05:42:51,512 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 571393ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 05:42:51,657 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 05:53:05,521 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 05:53:05,540 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 571613ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 06:03:16,558 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 570275ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 06:03:16,516 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 06:13:30,441 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 572724ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 06:13:30,504 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 06:23:40,554 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 06:23:40,603 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 570192ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 06:33:52,558 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 570833ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 06:33:52,502 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 06:43:48,482 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 576265ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 06:43:48,601 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 06:54:02,675 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 571496ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 06:54:02,521 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 07:04:14,451 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 570927ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 07:04:14,515 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 07:14:27,540 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 07:14:27,486 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 572096ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 07:24:35,535 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 07:24:35,406 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 570291ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 07:34:52,420 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 571938ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 07:34:52,414 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 07:45:03,748 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 07:45:03,805 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 571036ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 07:55:14,527 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 07:55:14,538 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 570384ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 08:05:27,530 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 571862ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 08:05:27,554 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 08:15:39,526 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 08:15:39,542 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 572296ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 08:25:50,517 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 08:25:50,377 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 572143ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 08:36:03,198 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 08:36:03,683 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 570340ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 08:38:16,532 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 111896ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 08:38:16,620 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 08:47:40,072 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 563295ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 08:57:46,011 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 566176ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 08:57:46,007 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 09:07:53,066 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 565456ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 09:07:53,008 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 09:18:01,014 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 566161ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 09:18:01,008 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 09:28:08,012 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 565396ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 09:28:08,008 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 09:36:48,066 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 479072ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 09:36:48,006 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 09:49:01,018 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 732940ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 09:58:26,023 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 126528ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 09:58:26,014 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 10:32:42,535 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 10:32:42,548 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 96101ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 11:59:21,075 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 11:59:21,079 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 1215164ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 12:16:05,073 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 943903ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 12:16:05,066 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 13:12:52,084 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 13:12:52,108 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 265723ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 13:21:40,106 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 247198ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 13:21:40,085 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 15:16:31,385 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 129818ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 15:16:31,337 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 18:36:20,314 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 1207642ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 18:36:20,429 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 18:48:55,006 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 18:48:55,011 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 308989ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 18:53:56,064 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 300939ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 19:00:33,011 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 296929ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 19:00:33,005 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 19:05:34,010 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 300938ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 19:12:11,067 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 297048ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 19:12:11,005 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 19:17:12,009 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 300937ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 19:23:49,065 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 296981ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 19:23:49,005 ERROR 92266 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 19:26:50,027 ERROR 89138 nodejs.ClusterClientNoResponseError: client no response in 180944ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 89138
hostname: guchi.local

2022-01-12 22:27:27,185 ERROR 48169 nodejs.ClusterClientNoResponseError: client no response in 969492ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 48169
hostname: guchi.local

2022-01-12 22:27:27,165 ERROR 48173 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
2022-01-12 22:32:41,030 ERROR 48169 nodejs.ClusterClientNoResponseError: client no response in 294750ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout._onTimeout (/Users/dasouche/Desktop/eggDemo/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (internal/timers.js:555:17)
    at processTimers (internal/timers.js:498:7)
name: "ClusterClientNoResponseError"
pid: 48169
hostname: guchi.local

2022-01-12 22:32:41,015 ERROR 48173 [egg-mongoose] mongodb://localhost:27017/noteDemo disconnected
